app/src/main/java/nodomain/freeyourgadget/gadgetbridge/service/devices/miband/MiBandSupport.java

```diff
@@ -276,7 +279,7 @@ private MiBandSupport setFitnessGoal(TransactionBuilder transaction) {
             transaction.write(characteristic, new byte[]{
                     MiBandService.COMMAND_SET_FITNESS_GOAL,
                     0,
-                    (byte) (fitnessGoal  &  0xff),
+                    (byte) (fitnessGoal & 0xff),
```

Comment:
Umm, sorry. I could swear the diff showed me something different (change from (fitnessGoal >>> 8) & 0xff) to fitnessGoal & 0xff.

Never mind, then.